### DISCUSSION TITLE
billboards with mesh instancer

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#462] [Feature Requests]
not sure if this is possible.

using mesh instancer i would like all the instances to point to the camera, as if they were billboarded 
seems like it should be possible?

**Comment by pandrr** [comment]
billboarded mesh instancer is basically the pointcloud ops, right ?

or do you really need 3d mesh

**Comment by marioepsley** [comment]
I did try using the point cloud op but I found the points wouldn't go big enough for what i needed ;) I am just using images so it would work, but yeah 3d meshes would be great as well ;)
